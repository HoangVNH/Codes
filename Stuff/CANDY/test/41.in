10
86
XXVXVDDVVXVVDVVVVVXXVVXDXDVXDDVXVXDDXVDXXDXXVXVVVVDXXVVVDXDDXXVXVDXDXXVVDDVVVDVXXXDVVD
82
VVVDXDVDXDVDXDVXDXDVXDDVVXXDXVDDDVDVDVXVVXDXVVXXDXVDXVXVDXVXXVVVVXXDXVXXXDXDXVXDVV
94
VXXVXXVVVDXDVXVDXVVXVDVXDXDDXXXVVDVDXDXDXXVDXDVVDXXXXVXXVDDDXXXDVDVXDXVDVVVXDVDVDDDVVDVVVDDVVX
84
VXXDVXVDDDDXDVXDDVDVVVDVXVVVVVVXDXVDDDXDDXDDDVXXXVDVXXVVXDVXDVDXVVXXVVXVVDDDVXXVDDXX
87
VXDXVDDXXDXVVVXVDDDVXDDDVVXXVXDVDVXDDVXVVDDDXDDVDXDDVXVVXVDDDDDDDVXDDXXXDXDDDDXDVXDXDDV
79
XDXDVVXXDVXVDVVXDDXVDXVXDVVDXDVDVXDXVVVVVVXVVVVDXDXXVXDDVDVVDVVVDXXXVXVDVDDVVXV
70
XDXXDVVDXDVXVXVDXXXXVDVDXXXVXVDXDDXVXVXDVVDVXVXXDXXVDDDDVDDXDVDVVDVVVV
83
XXXDVDXVVXXDDDXXVVVDVXDXXVDVXVVVDVDDDVXVVVXVVDXVXDXVDVXXXVDXXVVDVXXDXDXDDVXVDDXXVDV
81
VVDVXVXXXXDVXXDVDDDDXXVXVXXXDDDDXXXDVXVDVXXDVXXXXVDDDDDXVVDDDVDVXXVDVDXDDVXXVDXDD
67
VXXXVVVDXXDVVDDDDXXDVDXDVDXVXDVVXXDDDDDXVDXVDVVVXVDVXDVXVDXVXXDVDXV
