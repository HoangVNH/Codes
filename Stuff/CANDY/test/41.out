85
81
93
83
86
78
68
82
80
65
